lib/modules/manager/poetry/extract.ts

```diff
@@ -54,6 +54,8 @@ function extractFromSection(
       continue;
     }
 
+    let pep503NormalizeRegex = /[-_.]+/g;
```

Comment:
```suggestion
    let pep503NormalizeRegex = regEx(/[-_.]+/g);
```
use our regEx wrapper